src/encoded/tests/test_ingestion_listener.py

```diff
@@ -8,7 +8,8 @@
 from dcicutils.qa_utils import ignored, notice_pytest_fixtures
 from uuid import uuid4
 from pyramid.testing import DummyRequest
-from ..ingestion_listener import IngestionQueueManager, run, IngestionListener, verify_vcf_file_status_is_not_ingested
+from ..ingestion_listener import (IngestionQueueManager, run, IngestionListener,
+                                  verify_vcf_file_status_is_not_ingested, IngestionError, IngestionReport)
```

Comment:
When you flow onto multiple lines, i suggest starting names on the second line because it will backdent farther and you will get much less needs for more continuation lines after that.
```suggestion
from ..ingestion_listener import (
    IngestionQueueManager, run, IngestionListener, verify_vcf_file_status_is_not_ingested,
    IngestionError, IngestionReport,
)
```